cpp/set_property_module/set_property_module.cpp

```diff
@@ -0,0 +1,296 @@
+#include <map>
+#include <unordered_set>
+
+#include <fmt/core.h>
+#include <mgp.hpp>
+
+#include "mgclient.hpp"
+
+constexpr static std::string_view kResult = "result";
+constexpr static std::string_view kQueryExecuted = "queryExecuted";
+constexpr static std::string_view kSourceProperties = "sourceProperties";
+constexpr static std::string_view kSourceVariable = "sourceVariable";
+constexpr static std::string_view kSourceNode = "sourceNode";
+constexpr static std::string_view kSourceRel = "sourceRel";
+constexpr static std::string_view kTargetProperties = "targetProperties";
+constexpr static std::string_view kTargetVariable = "targetVariable";
+constexpr static std::string_view kTargetNode = "targetNode";
+constexpr static std::string_view kTargetRel = "targetRel";
+
+constexpr char *kMgHost = "MG_HOST";
+constexpr char *kMgPort = "MG_PORT";
+constexpr char *kMgUsername = "MG_USERNAME";
+constexpr char *kMgPassword = "MG_PASSWORD";
+
+constexpr char *kDefaultHost = "localhost";
+constexpr uint16_t kDefaultPort = 7687;
+
+mg::Client::Params GetClientParams() {
+  auto *host = kDefaultHost;
+  auto port = kDefaultPort;
+  auto *username = "";
+  auto *password = "";
+
+  auto *maybe_host = std::getenv(kMgHost);
+  if (maybe_host) {
+    host = std::move(maybe_host);
+  }
+
+  const auto *maybe_port = std::getenv(kMgPort);
+  if (maybe_port) {
+    port = static_cast<uint16_t>(std::move(*maybe_port));
+  }
+
+  const auto *maybe_username = std::getenv(kMgUsername);
+  if (maybe_username) {
+    username = std::move(maybe_username);
+  }
+
+  const auto *maybe_password = std::getenv(kMgPassword);
+  if (maybe_password) {
+    password = std::move(maybe_password);
+  }
+
+  return mg::Client::Params{.host = std::move(host),
+                            .port = std::move(port),
+                            .username = std::move(username),
+                            .password = std::move(password)};
+}
```

Comment:
mg::Client::Params accepts string and you are moving pointer.  I think correct solution is following
```suggestion
  return mg::Client::Params{.host = std::string(host),
                            .port = port,
                            .username = std::string(username),
                            .password = std::string(password)};
}
```